runtime/common/src/main/kotlin/kotlinx/serialization/json/JsonBuilder.kt

```diff
@@ -1,102 +0,0 @@
-/*
```

Comment:
Bring back accidentally renamed `JsonObjectBuilder.kt` since it also contains `JsonArrayBuilder`